aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -41,6 +41,18 @@ void check_tensor_memory_format(const Tensor& ref, const Tensor& other) {
 //
 // Please read the README.md in this directory before editing this file
 
+bool all_inputs_sharing_scale_and_zero_point(
+    const c10::List<Tensor>& qxs,
+    double scale,
+    int64_t zero_point) {
+  bool is_valid = true;
+  for (const auto i : c10::irange(qxs.size())) {
+    is_valid |= qxs[i].q_scale() == scale;
```

Comment:
if `is_valid` is initialized as true, then any further `|=` will not change its value. Did you mean `&=`?

Reply:
Ohhh, yes! Fixed.